llvm/test/SYCLLowerIR/addrspacecast_handling.ll

```diff
@@ -0,0 +1,83 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; RUN: opt < %s -LowerWGScope -S | FileCheck %s
+
```

Comment:
Please add a comment what this test checks.

Reply:
Added comment.